public/data/abdullahsajid-ma.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Abdullah Sajid",
+  "type": "personal",
+  "bio": "Learning Open Source",
+  "avatar": "https://github.com/abdullahsajid-ma.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/abdullahsajid-ma",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/aabdullahsajid",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/aabdullahsajid/",
+      "icon": "instagram"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

Comment:
I recommend capitalizing the `i` here. What do you think? 🙂
```suggestion
      "name": "Follow me on LinkedIn",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant